pkg/plugins/optional/grafana/v1alpha/scaffolds/edit.go

```diff
@@ -41,15 +49,121 @@ func (s *editScaffolder) InjectFS(fs machinery.Filesystem) {
 	s.fs = fs
 }
 
+func fileExist(configFilePath string) bool {
```

Comment:
```suggestion
func hasFileWith(configFilePath string) bool {
```
methods that return boolean usually are prefixed with has or is